scripts/build_python.sh

```diff
@@ -190,7 +190,7 @@ if [ "$install_pytest_requirements" = "yes" ]; then
 fi
 
 if [ -n "$extra_packages" ]; then
-    WHEEL+=("$extra_packages")
+    WHEEL+=($extra_packages)
```

Comment:
looks good, however I believe restyler shellharden fights you (and probably what got this to be wrong in the first place...).

Ideally we would find some other way that works, but if that fails we can mark this in ignore at https://github.com/project-chip/connectedhomeip/blob/master/.restyled.yaml#L41

Please try to see if you find a method that both works and makes shellharden happy ... and if that fails lets add the exception.

Reply:
Yes I have seen that, I will look into this tomorrow.

Reply:
Changed the script so that extra_packages can be specified multiple times and uses an array. I was tempted to rename the option to --extra_package in order to confer the change in meaning but that could break for ppl already using it.